src/rules/converters/no-implicit-dependencies.ts

```diff
@@ -0,0 +1,23 @@
+import { RuleConverter } from "../converter";
+
+export const convertNoImplicitDependencies: RuleConverter = tslintRule => {
+    const ruleArgs = [];
+
+    tslintRule.ruleArguments.forEach(element => {
+        if (element === "dev") {
+            ruleArgs.push({ devDependencies: false });
+        } else if (element === "optional") {
+            ruleArgs.push({ optionalDependencies: false });
+        } else {
+            ruleArgs.push({ devDependencies: element });
+        }
+    });
+    return {
+        rules: [
+            {
+                ruleName: "import/no-extraneous-dependencies",
+                ...(ruleArgs.length > 0 && { ruleArguments: ruleArgs }),
+            },
+        ],
```

Comment:
I am pretty sure you need to provide the plugin after rules array as it is being handled by a plugin. Check [eslint plugin Roadmap](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/ROADMAP.md) for more info.